packages/components/psammead-radio-schedule/src/__snapshots__/index.test.jsx.snap

```diff
@@ -1846,11 +1827,6 @@ exports[`RadioSchedule should render rtl radio schedules correctly 1`] = `
             >
               NEXT
             </span>
-            <span
-              class="c13"
-            >
-              ,
```

Comment:
Looking at the accessibility notes 5,6 and 7 on [these designs ](https://app.zeplin.io/project/5c8a5e021d454124ef90669f/screen/5dc0724850731722a4aa767a), the visually hidden text should be `[State], [Brand title], [00:00], Episode [Episode title]`. Removing this comma will cause the `state` to be read in one breath with the `Brand title`.

Reply:
Good point, done now